build.sbt

```diff
@@ -1,8 +1,8 @@
-name := """stream-example"""
+name := "stream-example"
```

Comment:
The activator for some reason likes triple-quotes on project names. I wonder why.